### PR TITLE
Slight addition to handle KGCL `RemoveTextDefinition`

### PR DESCRIPTION
**Pull request by hrshdhgd**
Here, I'm replacing the definition entirely to "".

Example command:
```
remove definition for FBbt:00000001
```

Implements https://github.com/INCATools/kgcl/pull/28

### CODE REVIEW

**Review by cmungall — CHANGES_REQUESTED** [review]
To implement RemoveTextDefinition, add another switch in the code to test if the change instantiations RemoveTextDefinition

**Comment by hrshdhgd** [comment]
> To implement RemoveTextDefinition, add another switch in the code to test if the change instantiations RemoveTextDefinition

Done!

File: src/oaklib/implementations/simpleobo/simple_obo_implementation.py [hunk 1]
```diff
@@ -813,8 +813,12 @@ def apply_patch(
             t = self._stanza(patch.about_node, strict=True)
             for tv in t.tag_values:
                 if tv.tag == TAG_DEFINITION:
-                    # The strip() portion is to handle a KGCL bug
-                    tv.replace_quoted_part(patch.new_value.strip("'"))
+                    if isinstance(patch, kgcl.RemoveTextDefinition):
```
Comment (cmungall) [inline src/oaklib/implementations/simpleobo/simple_obo_implementation.py:816]: this should be at the same level as the other switches

**Review by cmungall — CHANGES_REQUESTED** [review]

File: src/oaklib/implementations/simpleobo/simple_obo_implementation.py (hunk 1 above)
Comment (hrshdhgd) [reply src/oaklib/implementations/simpleobo/simple_obo_implementation.py]: Done

File: src/oaklib/implementations/simpleobo/simple_obo_implementation.py [hunk 2]
```diff
@@ -809,11 +809,16 @@ def apply_patch(
             t = self._stanza(patch.about_node, strict=True)
             t.add_quoted_tag_value(TAG_DEFINITION, patch.new_value.strip("'"), xrefs=[])
             modified_entities.append(patch.about_node)
+        elif isinstance(patch, kgcl.RemoveTextDefinition):
+            t = self._stanza(patch.about_node, strict=True)
+            for tv in t.tag_values:
+                if tv.tag == TAG_DEFINITION:
+                    # This is a remove_definition request
+                    tv.value = ""
```
Comment (cmungall) [inline src/oaklib/implementations/simpleobo/simple_obo_implementation.py:817]: the tag-val pair should be properly removed

**Review by cmungall — CHANGES_REQUESTED** [review]
see comments

File: src/oaklib/implementations/simpleobo/simple_obo_implementation.py (hunk 2 above)
Comment (hrshdhgd) [reply src/oaklib/implementations/simpleobo/simple_obo_implementation.py]: Done

**Review by cmungall — APPROVED** [review]

**Pull request merged**